clang/lib/DPCT/MapNames.cpp

```diff
@@ -27,6 +27,9 @@ std::string MapNames::getClNamespace(bool KeepNamespace, bool IsMathFunc) {
 std::string MapNames::getDpctNamespace(bool KeepNamespace) {
   return DpctNamespace[KeepNamespace];
 }
+std::string MapNames::getExpNamespace(bool KeepNamespace, bool IsMathFunc) {
+  return getClNamespace(KeepNamespace, IsMathFunc) + "ext::oneapi::experimental::";
```

Comment:
Currently no math functions in experimental, can use `false` directly.

Reply:
Done.